solr/solrj/src/test/org/apache/solr/client/solrj/impl/ClusterStateProviderTest.java

```diff
@@ -34,15 +35,20 @@
 import org.apache.solr.common.cloud.ClusterState;
 import org.apache.solr.common.cloud.DocCollection;
 import org.apache.solr.common.util.NamedList;
+import org.apache.solr.embedded.JettySolrRunner;
 import org.hamcrest.Matchers;
+import org.junit.After;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
 public class ClusterStateProviderTest extends SolrCloudTestCase {
 
+  private static JettySolrRunner jettyNode1;
+  private static JettySolrRunner jettyNode2;
```

Comment:
static fields in tests that refer to Solr should be null'ed out, so there's some burden to them.  Here... I think you could avoid these altogether and simply call `cluster.getJettySolrRunner(0)` which isn't bad!

Reply:
could add a convenience method if you like.